step-framework-parent/step-framework-collections/src/test/java/step/core/ql/OqlAttributesAwareFilterVisitorTest.java

```diff
@@ -0,0 +1,43 @@
+package step.core.ql;
+
+import org.antlr.v4.runtime.*;
+import org.junit.Assert;
+import org.junit.Test;
+import step.core.collections.Filter;
+import step.core.collections.filters.Equals;
+
+import java.util.Arrays;
+
+public class OqlAttributesAwareFilterVisitorTest {
```

Comment:
@lucianbaciu Code coverage of OQLAttributesAwareFilterVisitor (45% on lines) to be increased